packages/kbn-pm/dist/index.js

```diff
@@ -43781,6 +43781,18 @@ class KbnClientSavedObjects {
         this.log = log;
         this.requester = requester;
     }
+    /**
+     * Run the saved objects migration
+     */
+    async migrate() {
```

Comment:
TIL we need to manually generate and commit this file, and best part is it got mixed LF and CRLF line feeds.